pom.xml

```diff
@@ -207,7 +207,8 @@
         <version.org.jboss.xnio.xnio-nio>${version.org.jboss.xnio}</version.org.jboss.xnio.xnio-nio>
         <version.org.jmockit>1.39</version.org.jmockit>
         <version.org.mockito>2.18.0</version.org.mockito>
-        <version.org.mock-server.mockserver-netty>5.9.0</version.org.mock-server.mockserver-netty>
+        <version.org.bouncycastle>1.64</version.org.bouncycastle>
```

Comment:
Is the BC upgrade needed for the tests to pass with `mockserver-netty` 5.8.1?

Emmanuel has a PR that will update the `org.bouncycastle` version:

https://github.com/wildfly/wildfly-core/pull/4095

Reply:
The version upgrade is not needed, but it's needed the movement of BC to the main `pom.xml` (which is also done in the other PR). Checking the other PR maybe it's better if we wait until that PR is merged and work over it. I think with that PR I just need to upgrade the version of the mock-server and add BC to the ` testsuite/standalone/pom.xml`.

Reply:
+1

Reply:
@rmartinc The referenced PR has now been merged, I am going to mark this as needing a rebase for the steps you mentioned.

Reply:
OK, I rebased and did the same with the new poms. The mock-server is downgraded and BC is added in the two places the mock-server is used (now using the version managed by the main pom). Let's see the tests...

Reply:
@darranl @fjuma It is rebased and all the tests pass now. QA found the same in https://issues.redhat.com/browse/WFCORE-5060 (the new one is a duplicate, I'm asking to close it).